index.js

```diff
@@ -26,17 +26,18 @@ var prettyMs = require('pretty-ms')
 var tablify = require('tablify').tablify
 var sum = require('math-sum')
 var ncp = denodeify(require('ncp'))
+var yargs = require('yargs')
 temp.track()
 
 function getDeps () {
   return readFile('package.json', 'utf8').then(function (str) {
     var json = JSON.parse(str)
     var deps = extend({}, json.dependencies)
-    if (process.argv.indexOf('--production') === -1) {
+    if (!argv["production"]) {
```

Comment:
I made it like this to make sure it's consistent with what we have for `no-optional` flag. Because `argv.no-optional` is invalid.